### DISCUSSION TITLE
CSS: Hover parent element and target child element

### DISCUSSION THREAD

**Discussion by kaisiant** [chakra-ui/chakra-ui#3336] [Q&A] [answered]
Saw a similar question on #2386 but seem outdated.

My use case:
I want to hover the container and change the child element text color.
```
<Box> //container
 <Box> //child
 <Box/>
</Box>
```
How can I do that, I am using nextjs.

**Comment by chr-ge** [comment, marked as answer]
You can just use `_groupHover`:
```jsx
<Box padding='6' role="group">
  <Box _groupHover={{ color: "pink.100" }}>
    <Text>Change My Color</Text>
  </Box>
</Box>
```
You can place the `_groupHover` on the child `Box` or on the `Text`.
